ngi_reports/utils/entities.py

```diff
@@ -261,9 +270,18 @@ def populate(self, log, organism_names, **kwargs):
                             log.warn('Insufficient info "{}" for sample {}'.format('average_size_bp', sample_id))
                     else:
                         log.warn('No library validation step found {}'.format(sample_id))
-
+           
                 samObj.preps[prep_id] = prepObj
 
+            # exception for case of multi-barcoded sample from different preps run on the same fc (only if -b flag is set)
+            if kwargs.get('barcode_from_fc'):
+                listOfBarcodes = sum([all_barcodes.barcode for all_barcodes in list(samObj.preps.values())], [])
+                if len(list(dict.fromkeys(listOfBarcodes))) >= 1:
+                    listOfFlowcells = sum([all_flowcells.seq_fc for all_flowcells in list(samObj.preps.values())], [])
```

Comment:
I'd rename `listOfBarcodes` and `listOfFlowcells`, since camel case is usually used for objects. Maybe `list_of_barcodes` or simply `barcodes`?